src/features/movies/MoviePage/Movie/components/MovieTile/styled.js

```diff
@@ -88,6 +88,10 @@ export const MetaWrapper = styled.div`
     display: flex;
     flex-direction: column;
     gap: 8px;
+
+    @media (max-width: ${({ theme }) => theme.breakpoints.mobileMax}px) {
+        gap: 4px;
+    }
 `;
 export const MetaContainer = styled.div`
```

Comment:
Konrad, no space between 95, 96 lines.
Do we need MetaContainer because is empty?

Reply:
I added space between lines. 
I chacked it and yes, we need MetaContainer for Flex.